FlagsmithClient/src/main/java/com/flagsmith/internal/FlagsmithRetrofitService.kt

```diff
@@ -58,9 +60,9 @@ interface FlagsmithRetrofitService {
             val client = OkHttpClient.Builder()
                 .addInterceptor(envKeyInterceptor(environmentKey))
                 .let { if (cacheConfig.enableCache) it.addNetworkInterceptor(cacheControlInterceptor()) else it }
-                .callTimeout(cacheConfig.requestTimeoutSeconds, java.util.concurrent.TimeUnit.SECONDS)
-                .readTimeout(cacheConfig.readAndWriteTimeoutSeconds, java.util.concurrent.TimeUnit.SECONDS)
-                .writeTimeout(cacheConfig.readAndWriteTimeoutSeconds, java.util.concurrent.TimeUnit.SECONDS)
+                .callTimeout(requestTimeoutSeconds, java.util.concurrent.TimeUnit.SECONDS)
+                .readTimeout(readAndWriteTimeoutSeconds, java.util.concurrent.TimeUnit.SECONDS)
+                .writeTimeout(readAndWriteTimeoutSeconds, java.util.concurrent.TimeUnit.SECONDS)
```

Comment:
I'm not entirely sure why we have grouped `readAndWriteTimeout` into a single attribute. It makes it a little confusing in my opinion. I think we should keep them separate. Appreciate it bloats the constructor a little bit but I feel like these will not be used very often at all and so we can just document them and probably forget about them...

Reply:
Yes was just trying to keep the constructor args down @matthewelwell

Reply:
Yeah, makes sense. Think I'd rather just add the extra argument to keep them as explicit as possible.

Reply:
Sorted